cfg/qt.cfg

```diff
@@ -3568,33 +3594,38 @@
       <function name="back" yields="item"/>
       <function name="last" yields="item"/>
       <function name="data" yields="buffer"/>
-      <function name="shrink_to_fit" action="change-internal"/>
-      <function name="squeeze" action="change-internal"/>
-      <function name="reserve" action="change-internal"/>
+      <function name="rbegin" yields="start-iterator"/>
+      <function name="crbegin" yields="start-iterator"/>
+      <function name="rend" yields="end-iterator"/>
+      <function name="crend" yields="end-iterator"/>
     </access>
   </container>
-  <container id="qtString" startPattern="QString" endPattern="" inherits="qtContainer" opLessAllowed="true">
+  <container id="qtString" startPattern="QString" endPattern="" inherits="qtContainer" opLessAllowed="true" itEndPattern="&gt; :: iterator|const_iterator|reverse_iterator|const_reverse_iterator">
```

Comment:
I do not believe `&gt;` should be used here

Reply:
Oops. Yes i am sure that is wrong, copy & paste error.